literature.bib

```diff
@@ -4129,7 +4206,7 @@ @inproceedings{HSMI:ICST14
 	doi = {10.1109/ICST.2014.43},
 	tt-tags = {SPL Testing},
 	tc-tags = {classified by Thomas,testing,test-case selection,sample-based analysis,optimized product-based analysis,implementation independent,family-based specification,requirements},
-	sampling-tags = {name=incremental heuristic,greedy,product set,test artifacts,requirements coverage,sampling efficiency,unavailable tool,evaluation,testing},
+	sampling-tags = {name=incremental heuristic,greedy,test artifacts,requirements coverage,sampling efficiency,effectiveness,unavailable tool,evaluation,testing},
```

Comment:
why that change? any evidence?

Reply:
For `product set` I removed it first as it is never mentioned as an available input in the classification or any other work. However, I readded it but moved it to the end of the list as it is listed under Further Tags on the website.

As for effectiveness, the approach evaluates how optimal a solution is. This is for me an indicator of the effectiveness of the approach. It is the same for coverage based approaches that evaluate how much of a certain t-wise coverage was achieved. But in this case, it evaluates how optimal the solution is by calculating the derivation. So I added the effectiveness tag.